src/ripple/protocol/impl/tokens.cpp

```diff
@@ -30,13 +30,22 @@
 
 namespace ripple {
 
-static char rippleAlphabet[] =
+static constexpr char const* const alphabetForward =
     "rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz";
 
-static char bitcoinAlphabet[] =
-    "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
+static constexpr std::array<int, 256> const alphabetReverse = []() constexpr
```

Comment:
Another point I don't feel strongly about, but the trailing `constexpr` is unnecessary.  As of C++17 any lambda that can be `constexpr` automatically is.  To quote from cppreference:

> constexpr: explicitly specifies that the function call operator is a constexpr function. When this specifier is not present, the function call operator will be constexpr anyway, if it happens to satisfy all constexpr function requirements

https://en.cppreference.com/w/cpp/language/lambda

Reply:
Yes i didn't know that change! Very nice.